dependencies.tsv

```diff
@@ -2,22 +2,22 @@ code.google.com/p/go.net	hg	937a34c9de13c766c814510f76bca091dee06028	239
 github.com/ajstarks/svgo	git	89e3ac64b5b3e403a5e7c35ea4f98d45db7b4518	2014-10-04T21:11:59Z
 github.com/juju/blobstore	git	3aa294463f563d40f3722a00a4c2de72dbbe3f76	2014-11-14T04:26:36Z
 github.com/juju/errors	git	036046bfdccf6f576e2e5dec7f7878597bcaebe7	2015-02-11T20:59:49Z
-github.com/juju/gojsonpointer	git	0154bf5a168b672d8c97d8dd83a54cb60cd088e8	2014-07-18T03:59:30Z
+github.com/juju/gojsonpointer	git	afe8b77aa08f272b49e01b82de78510c11f61500	2015-02-04T19:46:29Z
 github.com/juju/gojsonreference	git	f0d24ac5ee330baa21721cdff56d45e4ee42628e	2015-02-04T19:46:33Z
 github.com/juju/gojsonschema	git	e1ad140384f254c82f89450d9a7c8dd38a632838	2015-03-12T17:00:16Z
 github.com/juju/httpprof	git	14bf14c307672fd2456bdbf35d19cf0ccd3cf565	2014-12-17T16:00:36Z
-github.com/juju/jujusvg	git	0f22fd5a3521f2b4d5ca6283b24536762de0cc28	2015-04-01T21:47:01Z
+github.com/juju/jujusvg	git	64d25ad9d7f1c57c61d5b5e8baf7035d4e398026	2015-03-31T21:50:31Z
 github.com/juju/loggo	git	dc8e19f7c70a62a59c69c40f85b8df09ff20742c	2014-11-17T04:05:26Z
 github.com/juju/names	git	ce4ecb2967822062fc606e733919c677c584ab7e	2015-02-20T07:57:36Z
 github.com/juju/schema	git	27a52be50766490a6fd3531865095fda6c0eeb6d	2014-07-23T04:23:18Z
 github.com/juju/testing	git	b93c38205ac2e3817b7b1d9d60e89e69c99a6362	2015-03-16T13:08:43Z
 github.com/juju/txn	git	e02f26c56cfb81c7c1236df499deebb0369bd97c	2014-09-25T11:49:22Z
-github.com/juju/utils	git	a90aa2e02b9e7fe354ab816e05b1e0a77f27242d	2015-02-23T16:02:32Z
+github.com/juju/utils	git	4a323da2a94607b1c4987d636cfe3139ef3a8262	2015-03-18T23:27:02Z
 github.com/juju/xml	git	91535ba18a6afd756e38a40c91fea0ed8e5dbaa6	2014-12-04T14:59:31Z
-golang.org/x/crypto	git	4ed45ec682102c643324fae5dff8dab085b6c300	2015-01-12T22:01:33Z
+golang.org/x/crypto	git	c57d4a71915a248dbad846d60825145062b4c18e	2015-03-27T05:11:19Z
 gopkg.in/check.v1	git	64131543e7896d5bcc6bd5a76287eb75ea96c673	2014-10-24T13:38:53Z
 gopkg.in/errgo.v1	git	81357a83344ddd9f7772884874e5622c2a3da21c	2014-10-13T17:33:38Z
-gopkg.in/juju/charm.v5-unstable	git	7937e962118ef3d5b60e13f71ec112e65560d44a	2015-04-03T09:13:00Z
+gopkg.in/juju/charm.v5-unstable	git	f995aeed973b4136137fb139d8aef61e10d00c63	2015-03-30T20:28:21Z
```

Comment:
:-1: 
Going back in time here.

Maybe 

    git checkout dependencies.tsv; make deps ; pushd $GOPATH/src/github.com/juju/jujusvg ; git checkout 64d25ad9d7f1c57c61d5b5e8baf7035d4e398026 ; popd ; make create-deps

so that only jujusvg is updated?